forc-plugins/forc-client/src/cmd/deploy.rs

```diff
@@ -55,6 +55,9 @@ pub struct Command {
     /// Deprecated in favor of `--default-signer`.
     #[clap(long)]
     pub unsigned: bool,
+    /// Submit the deployment transaction(s) without waiting for execution to complete.
+    #[clap(long)]
+    pub submit_only: bool,
```

Comment:
Should we support this for `forc run` as well?

Reply:
I think it makes sense, can I add this on a following PR?

Reply:
You can, although it might make sense to do it here so you only need to update the docs once.

Reply:
Heyy @sdankel, I added the documentation in this commit https://github.com/FuelLabs/sway/pull/6294/commits/57becddf78286e20a99748767eb9e569cb310b74

Reply:
Regarding forc run, I am working in parallel to implement the `--submit-only` flag in a separate PR. Is that okay with you?